src-tauri/Cargo.toml

```diff
@@ -56,12 +56,9 @@ serde_json = "1.0.132"
 toml_edit = { version = "0.22.23", features = ["serde"] }
 thiserror = "1.0.68"
 
-# deps implicitly used by JJ, which need to be pinned to a version to fix errors
-gix-object = "0.47.0"
```

Comment:
I think this is not needed anymore. At least I haven't found any issues after removing it.